### PR TITLE
fix_twice_diff_error_in_attach_detach_disk

### PR DESCRIPTION
**Pull request by ZongqingQi**
Signed-off-by: zonqi <zonqi@redhat.com>

### CODE REVIEW

**Comment by ZongqingQi** [comment]
PR:
    When the test command is executed, the sharebale property
    is added to the options so that two hard drives can be
    successfully attach. In addition, by removing the QEMU version
    information in the corresponding cfg file, the value of
    the status_error is false, so that it will not be wrongly reported.

**Comment by ZongqingQi** [comment]
Results of execution the case:
[root@localhost tp-libvirt]# avocado run --vt-type libvirt --vt-machine-type i440fx virsh.attach_detach_disk.attach_disk.normal_test.twice_diff_target
JOB ID     : f8e81fe418920eb380a19e566a2476c12e0397f1
JOB LOG    : /root/avocado/job-results/job-2018-02-27T08.38-f8e81fe/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.attach_detach_disk.attach_disk.normal_test.twice_diff_target: PASS (79.73 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 83.03 s

File: libvirt/tests/cfg/virsh_cmd/domain/virsh_attach_detach_disk.cfg [hunk 1]
```diff
@@ -107,7 +107,7 @@
                     at_dt_disk_at_options = "--driver qemu --subdriver raw --persistent --cache writethrough"
                     at_dt_disk_dt_options = "--persistent"
                 - twice_diff_target:
-                    qemu_file_lock = '2.9.0'
+                    #qemu_file_lock = '2.9.0'
```
Comment (chunfuwen) [inline libvirt/tests/cfg/virsh_cmd/domain/virsh_attach_detach_disk.cfg]: remove this comment out line

File: libvirt/tests/src/virsh_cmd/domain/virsh_attach_detach_disk.py [hunk 2]
```diff
@@ -274,6 +274,11 @@ def acpiphp_module_modprobe(vm, os_type):
         vm_ref = ""
 
     if test_cmd == "attach-disk":
+
+        #Since lock feature is introduced in libvirt 3.9.0 afterwards, disk shareable options
+        #need be set if both of disks need be attached successfully in case
```
Comment (chunfuwen) [inline libvirt/tests/src/virsh_cmd/domain/virsh_attach_detach_disk.py]: correct comment as below:
need to be if disk attachment succeed in multiple times

**Pull request merged**